docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -212,11 +212,7 @@ Control whether or not to capture screenshots. Options include `'on'`, `'off'`,
 `alert` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/dsl/monitor.ts#L47-L54[`AlertConfig`])::
 Enable or disable alerts. Read more about alerts in <<synthetics-settings-alerting>>.
 +
-`status.enabled` (`boolean`):::
-Enable monitor status alerts.
-+
-`tls.enabled` (`boolean`):::
-Enable TLS certificate alerts.
+`alert: {status: {enabled: [true|false]}}`
```

Comment:
nit: can we also also add tls to the fields?

Reply:
@vigneshshanmugam there isn't a TLS option for browser monitor though is there?

Reply:
It is possible in lightweight monitors, yes, but we don't do TLS validation for browser monitors